src/test/java/net/snowflake/ingest/streaming/internal/FlushServiceTest.java

```diff
@@ -110,7 +110,11 @@ private abstract static class TestContext<T> implements AutoCloseable {
       client = Mockito.mock(SnowflakeStreamingIngestClientInternal.class);
       Mockito.when(client.getParameterProvider()).thenReturn(parameterProvider);
       Mockito.when(client.getInternalParameterProvider()).thenReturn(internalParameterProvider);
-      storageManager = Mockito.spy(new InternalStageManager<>(true, "role", "client", null));
+      storageManager =
```

Comment:
please add unit and regression tests for the new endpoint

Reply:
We don't have UT & IT for all endpoints, it was added in this [JIRA](https://snowflakecomputing.atlassian.net/browse/SNOW-1490151) when doing endpoint wrapper refactor. Will be added once the server side change is release.

Reply:
Ideally test should go in with the code together, so we should wait for server side change to be released before merging this.